<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Glumboi's Trivia</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*===========================================================================
 * Colors
 *===========================================================================*/

/*===========================================================================
 * QWidget
 *===========================================================================*/
QWidget 
{
    background: palette(window);
    font-family: &quot;Segoe UI&quot;;
    font-size: 14px;
}


/*===========================================================================
 * QPushButton
 *===========================================================================*/
QPushButton
{
	background: palette(button);
    border: 2px solid transparent;
    padding-left: 14px;
    padding-right: 14px;
    padding-top: 4px;
    padding-bottom: 4px;
}

QPushButton:hover
{
    border: 2px solid rgba(0, 0, 0, 102);
}

QPushButton:pressed
{
    border: 2px solid transparent;
    background: rgba(0, 0, 0, 102);
}

QPushButton:checked:!pressed:enabled
{
	background: rgb(0, 99, 177);
    color: rgb(255, 255, 255);
}

QPushButton:checked:!pressed:hover:enabled
{
	border: 2px solid  rgb(0, 0, 0);
}



/*===========================================================================
 * QToolButton
 *===========================================================================*/
QToolButton
{
	background: transparent;
    border: none;
    padding-left: 16px;
    padding-right: 16px;
    padding-top: 6px;
    padding-bottom: 6px;
}

QToolButton:hover
{
	background: palette(button);
}

QToolButton:pressed
{
	  background: palette(dark);
}

QToolButton:checked:!pressed:enabled
{
	  background: rgb(0, 99, 177);
      color: rgb(255, 255, 255);
}


/*===========================================================================
 * QRadioButton
 *===========================================================================*/
QRadioButton::indicator
{
	border: none;
    width: 20px;
    height: 20px;
    image-position: center;
    image: url(:/style/images/radiobutton.svg);
}

QRadioButton::indicator:disabled
{
    image: url(:/style/images/radiobutton_disabled.svg);
}

QRadioButton::indicator:hover
{
    image: url(:/style/images/radiobutton_hover.svg);
}

QRadioButton::indicator:checked
{
    image: url(:/style/images/radiobutton_checked.svg);
}

QRadioButton::indicator:checked:disabled
{
    image: url(:/style/images/radiobutton_checked_disabled.svg);
}


QRadioButton::indicator:checked:hover
{
    image: url(:/style/images/radiobutton_checked_hover.svg);
}


/*===========================================================================
 * QComboBox
 *===========================================================================*/
QComboBox
{
    border: 2px solid rgba(0, 0, 0, 102);
    padding-left: 10px;
    padding-right: 10px;
    padding-top: 3px;
    padding-bottom: 4px;
}


QComboBox:hover:!pressed
{
    border-color: rgb(102, 102, 102);
}

QComboBox::pressed
{
    background: rgb(204, 204, 204);
}

QComboBox QAbstractItemView
 {
    border: 1px solid rgb(204, 204, 204);
    background: rgb(242, 242, 242);
    selection-background-color: lightgray;
   padding-left: 9px;
   padding-right: 9px;
}

QComboBox QAbstractItemView::item
{
	padding-top: 3px;
    padding-bottom: 4px;
}


QComboBox::drop-down
{
    border: none;
    padding-right: 8px;
}

QComboBox::down-arrow
{
    image: url(:/style/images/checkbox_down_arrow.svg);
    height: 6px;
    width: 12px;
}


/*===========================================================================
 * QCheckBox
 *===========================================================================*/
.QCheckBox::indicator
{
	width: 16px;
    height: 16px;
    border: 2px solid rgb(0, 0, 0);
}


.QCheckBox::indicator:disabled
{
    border: 2px solid rgb(172, 172, 172);
}


.QCheckBox::indicator::pressed
{
    border: 2px solid rgb(102, 102, 102);
    background: rgb(102, 102, 102);
}

.QCheckBox::indicator:checked
{
    image: url(:/style/images/checkbox_checked.svg);
}

.QCheckBox::indicator:checked:!pressed
{
    border: 2px solid rgb(0, 99, 177);
    background: rgb(0, 99, 177);
    image: url(:/style/images/checkbox_checked.svg);
}

.QCheckBox::indicator:checked:!pressed:hover
{
    border: 2px solid rgb(0, 0, 0);
}


.QCheckBox::indicator:indeterminate
 {
    image: url(:/style/images/checkbox_indeterminate.svg);
    border: 2px solid rgb(0, 99, 177);
}


/*===========================================================================
 * CToggleSwitch
 *===========================================================================*/
CToggleSwitch
{
    min-height: 24px;
}


CToggleSwitch::indicator
{
   image: url(:/style/images/toggleswitch_off.svg);
   image-position: center;
   width: 44px;
   height: 24px;
   padding: 0px;
}


CToggleSwitch::indicator::checked
{
   image: url(:/style/images/toggleswitch_on.svg);
   image-position: center;
}


/*===========================================================================
 * QToolBar
 *===========================================================================*/
 QMenuBar
 {
    background: palette(midlight);
    padding-bottom: 2px;
 }


QMenuBar::item
{
   background: none;
   border: none;
   padding-bottom: 10px;
   padding-top: 11px;
   padding-left: 12px;
   padding-right: 12px;
}


QMenuBar::item:selected
{
   background: rgba(0, 0, 0, 30);
   border: none;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_Genre">
      <property name="title">
       <string>GroupBox</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Stats</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_Reset">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Points: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Points">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Correctly answered:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Correct">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Skipped and wrongly answered Questions: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Skipped">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Question</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextEdit" name="textEdit_Question">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Possible Answers</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_Answers"/>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>Actions</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="pushButton_Skip">
            <property name="text">
             <string>Skip (-1 points)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Verify">
            <property name="text">
             <string>Verify Answer</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>40</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
